webpack-plugins/inject-leaflet-loader.js

```
@@ -0,0 +1,246 @@
+/**
+ * Webpack plugin to inject Leaflet loader snippet into HTML files
+ * Only for echarts-only builds
+ */
+const path = require("path");
+const fs = require("fs");
+const crypto = require("crypto");
+
+let _leafletMeta;
```

ESLint

[error] Unexpected dangling '_' in '_leafletMeta'.

(no-underscore-dangle)